src/encoded/static/components/genome_browser.js

```diff
@@ -529,6 +529,7 @@ class GenomeBrowser extends React.Component {
     }
 
     makeTrackLabel(file) {
+        console.log('FILE %o', file);
```

Comment:
Just wanted to check, do we want to print this?

Reply:
Oops; no. That was to diagnose something unrelated.